### PR TITLE
[hailctl] Look up the bucket to get VEP data from

### LINKED ISSUE #8078: hailctl dataproc submit does not support `--region` flag
Using hail 0.2.32, `hailctl dataproc submit` results in the following error:

```
hailctl dataproc submit hail-test {python file} -- {arguments to script}
Submitting to cluster 'hail-test'...
gcloud command:
gcloud dataproc jobs submit pyspark {python file} \
    --cluster=hail-test \
    --files= \
    --py-files=/var/folders/7y/hvrzyxts3xg74r3m2jbq0kc0zt3g3z/T/pyscripts_srh2ze4a.zip \
    --properties= \
    -- \
    {arguments to script}
ERROR: (gcloud.dataproc.jobs.submit.pyspark) The required property [region] is not currently set.
It can be set on a per-command basis by re-running your command with the [--region] flag.

You may set it for your current workspace by running:

  $ gcloud config set dataproc/region VALUE

or it can be set temporarily by the environment variable [CLOUDSDK_DATAPROC_REGION]
Traceback (most recent call last):
  File "/Users/aarong/Documents/gtex-wgs/.devenv/bin/hailctl", line 8, in <module>
    sys.exit(main())
  File "/Users/aarong/Documents/gtex-wgs/.devenv/lib/python3.7/site-packages/hailtop/hailctl/__main__.py", line 94, in main
    cli.main(args)
  File "/Users/aarong/Documents/gtex-wgs/.devenv/lib/python3.7/site-packages/hailtop/hailctl/dataproc/cli.py", line 107, in main
    jmp[args.module].main(args, pass_through_args)
  File "/Users/aarong/Documents/gtex-wgs/.devenv/lib/python3.7/site-packages/hailtop/hailctl/dataproc/submit.py", line 78, in main
    check_call(cmd)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError
``` 

However, adding `--region us-central1` to any location in the argument string to hailctl dataproc submit results in the argument being picked up as an input to the script, not to the underlying gcloud command

### PR DESCRIPTION
**Pull request by johnc1231**
So far there is still only one bucket, but here's the proposed restructuring. I'll make a follow up PR and duplicate the data to EU (we are bound to have European veppers) and London (since we know Sanger wants to use it). 

Also fixes #8078

### CODE REVIEW

**Comment by tpoterba** [comment]
sweet!

**Comment by tpoterba** [comment]
general structure looks great.

File: hail/python/hailtop/hailctl/dataproc/start.py [hunk 1]
```diff
@@ -148,12 +157,13 @@ def main(args, pass_through_args):
 
     # add VEP init script
     if args.vep:
-        allowed_buckets_list = args.requester_pays_allow_buckets.split(",") if args.requester_pays_allow_buckets else []
-        if not (args.requester_pays_allow_all or ('hail-us-vep' in allowed_buckets_list)):
-            raise RuntimeError("Need to enable requester pays on bucket 'hail-us-vep' to use vep. See --requester-pays-allow-all and --requester-pays-allow-buckets \
-                WARNING: This is likely prohibitively expensive to run on a cluster outside of the GCP 'us' region. If you need to run VEP outside the US, please contact the Hail team on https://discuss.hail.is for support.")
+        # VEP is too expensive if you have to pay egress charges. We must choose the right replicate.
+        project_region = sp.check_output(['gcloud', 'config', 'get-value', 'dataproc/region']).decode().strip()
```
[the remark this answers is not in the document]
Comment (johnc1231) [reply hail/python/hailtop/hailctl/dataproc/start.py]: Yep, good call.

**Comment by johnc1231** [comment]
Added a region argument, and mandated that users have either configured a region or are using `--region`. I believe older versions of gcloud assume the region is the default region for your project if you don't have anything specified, which causes users to have random errors when they update gcloud that they ask about on Zulip. This way, everyone gets a good error message

**Comment by tpoterba** [comment]
needs bump

**Pull request merged**